playbooks/roles/ocp-cli-plugins/defaults/main.yml

```diff
@@ -0,0 +1,9 @@
+
+# Mapping dict between input cli-plugin name and its equivalent cli-plugin module
+
+cli_plugin_map:
+  custom: "custom-cli-plugin.yml"
+  sample: "sample-cli-plugin.yml"
```

Comment:
This is no longer needed

Reply:
Removed sample-cli name from plugin to module mapping.